Feature/remove hardcoded (#17)
FXinnovation/alertmanager-webhook-servicenow: README.md

The webhook config is done in `config/servicenow.yml`.

All `default_incident` properties supports Go templating with structure defined in [AlertManager documentation](https://prometheus.io/docs/alerting/notifications/#data).

> Every key value pair in `default_incident` is sent as attributes of the incident created.

Reviewer: ```suggestion
Every key value pair in `default_incident` is sent as an incident field.
```
